README.md

```diff
@@ -90,11 +91,18 @@ controller.add(&hisThread);
 controller.add(&sensorReadings);
 ...
 ```
+or
+```c++
+// Instantiate a new StaticThreadController
```

Comment:
"Instantiate a new StaticThreadController with the number of threads to be supplied"